fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java

```diff
@@ -47,8 +47,9 @@
     implements CameraPermissionViewModel, QRCodeScanningViewModel {
 
   public static final String TAG = HomeViewModel.class.getSimpleName();
-  public static final String SCAN = "SCAN";
-  public static final String REQUEST_CAMERA_PERMISSION = "REQUEST_CAMERA_PERMISSION";
+
+  public enum HOMEVIEWACTION {SCAN, REQUEST_CAMERA_PERMISSION}
```

Comment:
`HOMEVIEWACTION` does not respect the usual naming convention.

A better name would be
```suggestion
  public enum HomeViewAction {SCAN, REQUEST_CAMERA_PERMISSION}
```